src/config_default.h

```diff
@@ -1122,6 +1122,17 @@
 #    endif
 #endif
 
+/**
+ * Debug file system command to read DHT sensor on the bus.
+ */
+#ifndef CONFIG_DHT_COMMAND_READ
+#    if defined(BOARD_ARDUINO_NANO) || defined(BOARD_ARDUINO_UNO) || defined(BOARD_ARDUINO_PRO_MICRO) || defined(CONFIG_MINIMAL_SYSTEM)
```

Comment:
I was not sure about those conditions - just copied them after _ds18b20_ above.

Reply:
These Arduino boards have very limited amount of memory, and lots of fs commands are disabled by default.